docs/hadoop.adoc

```diff
@@ -121,6 +167,187 @@ def defineGratefulDeadSchema(janusGraph) {
 }
 ----
 
+====== 2. Bulk loading into backend using Spark cluster in standalone mode.
+The steps followed above, under section *Bulk loading into backend using Local Spark* can be followed again to Bulk load data into Janusgraph using Spark.
+
+We will only need to modify our hadoop-bulk-load.properties to point to our Clustered spark, and not local.
+
+Also, in addition to that, we will need to distribute all the necessary Jars (In this case which comes pre-packaged with JanusGraph), so that required classes are loaded in required order and hence doesn't raise conflicts.
+
+[NOTE]
+We have copied all the jars under *janusgraph-distribution/lib* into /root/lib/janusgraph/ and the same directory structure is created across all workers, and jars are manually copied across all workers.
+
+[source, properties]
+----
+# hadoop-bulk-load.properties
+#
+# Hadoop Graph Configuration
+#
+gremlin.graph=org.apache.tinkerpop.gremlin.hadoop.structure.HadoopGraph
+gremlin.hadoop.graphReader=org.apache.tinkerpop.gremlin.hadoop.structure.io.gryo.GryoInputFormat
+gremlin.hadoop.graphWriter=org.apache.hadoop.mapreduce.lib.output.NullOutputFormat
+
+gremlin.hadoop.jarsInDistributedCache=true
+gremlin.hadoop.inputLocation=hdfs:///grateful-dead.kryo
+gremlin.hadoop.outputLocation=output
+gremlin.spark.persistContext=true
+
+#
+# SparkGraphComputer Configuration
+#
+spark.master=spark://127.0.0.1:7077
+spark.executor.memory=6g
+spark.executor.extraClassPath=/root/lib/janusgraph/*
+spark.serializer=org.apache.spark.serializer.KryoSerializer
+spark.kryo.registrator=org.apache.tinkerpop.gremlin.spark.structure.io.gryo.GryoRegistrator
+----
+
+[NOTE]
+The backend of choice here is Cassandra for all read example bellow. Additional configuration will be needed specific to that data store, if anything else is selected, like for example HBase, Scylla, etc.
+
+====== 3. Reads from backend data store into Hadoopgraph for OLAP against local spark
+
+The following properties file is used to load HadoopGraph into gremlin console, over which OLAP queries are done.
+
+[source, properties]
+----
+# hadoop-cassandra3.properties
```

Comment:
Use same properties file name that is packaged with the distribution, `read-cassandra-3.properties` because the user will expect to follow these directions line by line with the distribution.